packages/framework/aqueduct/src/container-runtime-factories/containerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -48,12 +48,9 @@ export class ContainerRuntimeFactoryWithDefaultDataStore extends BaseContainerRu
      * {@inheritDoc BaseContainerRuntimeFactory.containerInitializingFirstTime}
      */
     protected async containerInitializingFirstTime(runtime: IContainerRuntime) {
-        const router = await runtime.createRootDataStore(
-            this.defaultDataStoreName,
-            ContainerRuntimeFactoryWithDefaultDataStore.defaultDataStoreId,
+        await runtime.createRootDataStore(
+            this.defaultFactory.type,
+            defaultDataStoreId,
         );
-        // We need to request the data store before attaching to ensure it
```

Comment:
is this request removal change supposed to be part of this pr?

Reply:
This is not strictly related to this PR, but after PR https://github.com/microsoft/FluidFramework/pull/3822, this is not required for PureDataObjectFactory. Argument can be made that this is still required as a general concept, but I personally do not like it. There is nothing on contract for runtime interfaces saying that request for specific ("/") route needs to happen in order for initialization of data store to be done. I believe it's much better for implementers of data objects to do right thing here and ensure that factory's instantiateDataStore() does right things. And if DataObject is any guidance here, not doing right things causes more trouble down the road (see that PR for a list of issues) - doing requests on specific workflows is just hiding bugs.

let me know what you think - I can pull it into separate PR

Reply:
Sounds good, keeping seems fine then.  Wasn't sure if it accidentally got included and would break something